### ISSUE TITLE
Annotation tool: fails to load image on iPad

### ISSUE THREAD

**Issue by StephenChan** [beijbom/coralnet#473]
See this Google Groups thread; the problem started happening a few months ago: https://groups.google.com/g/coralnet-users/c/YiBzvY1nVx0

I don't have an iPad or iOS device handy myself, so I'd be looking to try replicating this on an iOS emulator.

**Comment by JiaCashon** [comment]
Aloha Stephen, I was hoping to work on CoralNet as I traveled on an iPad but found that the images wouldn’t pull up. Seems that the formatting is off. I’ve read your comment about doing the iOS emulator but not too sure how to go about it, is it something that I download? Sorry, not too techy and just hoping to work on this without a computer.

**Comment by StephenChan** [comment]
Hi Jia - since I don't personally have an iPad, iPhone, etc., the iOS emulator comment was actually my idea for encountering the annotation-tool bug myself, so I could investigate/fix it. Your goal would be to *not* encounter the bug though, which is the opposite!

Actually, a Linux or Windows virtual machine (or emulator) could be a potential workaround for this issue, but unfortunately I don't know how that might be set up on an iPad.

**Comment by JiaCashon** [comment]
Interesting and thanks for your response. I have been using CoralNet on my MacBook Air which operates under the MacOs and so does the Ipad I believe... I'll try switching to Linux or windows and see what happens.

**Comment by StephenChan** [comment]
A MacBook Air would be running macOS, while an iPad would be running... well, actually I just looked it up, and it's something called iPadOS which was branched off from iOS.

So macOS, iOS, and iPadOS are all different operating systems. This particular annotation tool bug hasn't been reported on macOS, but it has been on iOS (running on an iPhone) and iPadOS (running on an iPad).

**Comment by JiaCashon** [comment]
Yeah, I have a 9th generation ipad and apparently once you attach a
keyboard to some of the newer models, it flips into a mode more like MacOS
and may operate coralnet. I'm trying to contact Apple to see if that may be
the case with the 9th as well since it is an older model...let me know if
you want to know the outcome, otherwise I'll stop bothering you! Thank you
for the help you've given me this far, I appreciate it!

**Comment by StephenChan** [comment]
Ah interesting. Feel free to share the outcome, it might help other folks who have an iPad!

**Comment by JiaCashon** [comment]
I couldn't download an IOS emulator or MacOS to the ipad. Long story short,
Ipad's are not compatible and it looks like I am lugging my laptop around
Europe. Thanks for all of the help though!
